src/OAuth2/ResponseType/JwtAccessToken.php

```diff
@@ -127,5 +127,20 @@ protected function createPayload($client_id, $user_id, $scope = null)
             'token_type' => $this->config['token_type'],
             'scope'      => $scope
         );
+        
+        if (isset($this->config['jwt_extra_payload'])) {
+            if (is_callable($this->config['jwt_extra_payload'])) {
```

Comment:
@bshaffer I still don't like the idea that it is impossible to change values in $payload.